mw-webapp/src/pages/mainPage/MainPage.tsx

```diff
@@ -12,6 +13,16 @@ export const MainPage = () => {
         Hiii, Student!
       </h1>
       <Table />
+      {/* To test Sidebar component, will delete before merge */}
+      <Sidebar
+        trigger={
+          <button>
+            Open sidebar
+          </button>}
+        content={
+          <></>
```

Comment:
Cool, let's put the list with links to all pages in the appi the sidebar. 
Also let's add side bar to all other pages (maybe it will be better to put sidebar not in the page, but in the header, what do you think). You can create an issue for that  if it is needed,or if it is simple add it in current PR

Reply:
Created issue